FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2PersonaActivity.kt

```diff
@@ -58,93 +59,76 @@ class V2PersonaActivity : DemoActivity() {
+                            color = Color(0xFF2886DE)
+                        )
+                        Persona(
+                            person = person2,
+                            primaryText = person2.firstName + " " + person2.lastName,
+                            secondaryText = "Microsoft",
+                            onClick = {
+                                onClick(
+                                    person2.firstName,
+                                    context
                                 )
-                                Persona(
-                                    person = person3,
-                                    primaryText = person3.firstName + " " + person3.lastName,
-                                    secondaryText = "Microsoft",
-                                    tertiaryText = person3.status.toString(),
-                                    onClick = {
-                                        onClick(
-                                            person3.firstName,
-                                            coroutineScope,
-                                            scaffoldState
-                                        )
-                                    },
+                            },
+                        )
+                    }
+                }
+                item {
+                    Column {
+                        Text(
+                            modifier = Modifier.padding(start = 8.dp, top = 16.dp),
+                            text = "Three line Persona View with Xlarge Avatar",
+                            color = Color(0xFF2886DE)
+                        )
+                        Persona(
+                            person = person3,
+                            primaryText = person3.firstName + " " + person3.lastName,
+                            secondaryText = "Microsoft",
+                            tertiaryText = person3.status.toString(),
+                            onClick = {
+                                onClick(
+                                    person3.firstName,
+                                    context
                                 )
-                            }
-                        }
+                            },
+                        )
                     }
                 }
-
             }
         }
 
     }
 
-    fun onClick(text: String, coroutineScope: CoroutineScope, scaffoldState: ScaffoldState) {
-        scaffoldState.snackbarHostState.currentSnackbarData?.dismiss()
-        coroutineScope.launch {
-            val result = scaffoldState.snackbarHostState.showSnackbar(
-                message = "Clicked on $text"
-            )
-            when (result) {
-                SnackbarResult.ActionPerformed -> scaffoldState.snackbarHostState.currentSnackbarData?.dismiss()
-            }
-        }
+    fun onClick(text: String, context:Context) {
```

Comment:
This function has been added in DemoUtils. Remove this & refer the DemoUtils one.
Code Pointer :
https://github.com/microsoft/fluentui-android/blob/master/FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/util/DemoUtils.kt#L22